<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pbDone">
      <property name="text">
       <string>Готово</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="MainLayout">
      <item>
       <widget class="QFrame" name="framePhoto">
        <property name="styleSheet">
         <string notr="true">QFrame {
	border: 2px solid #757575;
}</string>
        </property>
        <layout class="QVBoxLayout" name="PhotoLayout">
         <item>
          <widget class="QLineEdit" name="lePhotoPath">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	border: 2px solid #757575;
	background-color: #424242;
	padding: 2px;
}</string>
           </property>
           <property name="text">
            <string>Путь к файлу</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblPhoto">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>600</height>
            </size>
           </property>
           <property name="text">
            <string>Фото</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbUploadPhoto">
           <property name="text">
            <string>Загрузить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameInfo">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
	border: 2px solid #757575;
}

QLabel {
	border: none;
}</string>
        </property>
        <layout class="QVBoxLayout" name="InfoLayout">
         <item>
          <layout class="QHBoxLayout" name="FullNameLayout">
           <item>
            <widget class="QLabel" name="lblFullName">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>ФИО</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leFullName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="CitizenshipLayout">
           <item>
            <widget class="QLabel" name="lblCitizenShip">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Гражданство</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leCitizenShip"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="PostLayout">
           <item>
            <widget class="QLabel" name="lblPost">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Должность</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePost"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="DateOfBirthLayout">
           <item>
            <widget class="QLabel" name="lblDateOfBirth">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Дата рождения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="deDateOfBirth">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>1800</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="GenderLayout">
           <item>
            <widget class="QRadioButton" name="rbMale">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>м</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbFemale">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>ж</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
